docs/conf.py

```diff
@@ -259,6 +262,23 @@
 nb_execution_show_tb = True
 nb_execution_timeout = -1
 nb_output_stderr = "remove"
+
+
+def pick_newtype_attrs(some_type: type) -> list:
+    return [
+        attr
+        for attr in dir(some_type)
+        if type(getattr(some_type, attr)) is typing.NewType
+    ]
+
+
```

Comment:
Could you move this one before all the variables (options) in this file? So right after the imports.